framework/basehandlers.py

```diff
@@ -380,6 +388,7 @@ def FlaskApplication(routes, pattern_base='', debug=False):
   """Make a Flask app and add routes and handlers that work like webapp2."""
 
   app = flask.Flask(__name__)
+  app.secret_key = "abc"
```

Comment:
I have a recent CL that securely generates and stores a different server-side secret.  I will write a CL to populate this flask session secret key as well.

Reply:
I now have this part waiting for review.
https://github.com/GoogleChrome/chromium-dashboard/pull/1285

Reply:
Thank you for adding this... I will merge it in this PR

Reply:
Done